src/components/Sections/VideoSection/__tests__/VideoSection.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { render } from "@testing-library/vue";
+import VideoSection from "..";
+
+const dummyId = "N0tRe4lYTID";
+const testYtUrl = "http://not-a-real-url.org/player/";
+
+describe("components/sections/video-section", () => {
+  it("renders an iframe with the expected src attribute", () => {
```

Comment:
We should include a test to ensure that the values of the passed parameters are encoded.

Reply:
won't do ;-) see comment on *supported parameters* before